src/components/TodoList.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { User } from '../types/User';
+import { Todo } from '../types/Todo';
+
+import './TodoList.scss';
+
+type Props = {
+  todos: Todo[];
+  users: User[];
+};
+
+export class TodoList extends React.PureComponent<Props> {
+  phoneNumber = (user: User) => (
+    user.phone.split('').filter(el => !Number.isNaN(+el)).slice(0, 11).join('')
+  );
+
+  visibleNumber = (str: string) => (
+    str.length > 10
+      ? `+${str[0]} (${str.slice(1, 4)}) ${str.slice(4, 7)} ${str.slice(7)}`
+      : `(${str.slice(0, 3)}) ${str.slice(3, 6)} ${str.slice(6)}`
+  );
```

Comment:
move such helper functions outside the component
```suggestion
  formatPhoneNumber = (user: User) => (
    user.phone.split('').filter(el => !Number.isNaN(+el)).slice(0, 11).join('')
  );

  getVisibleNumber = (str: string) => (
    str.length > 10
      ? `+${str[0]} (${str.slice(1, 4)}) ${str.slice(4, 7)} ${str.slice(7)}`
      : `(${str.slice(0, 3)}) ${str.slice(3, 6)} ${str.slice(6)}`
  );
```